app/assets/src/components/views/PipelineViz/PipelineViz.jsx

```diff
@@ -719,6 +729,49 @@ class PipelineViz extends React.Component {
     );
   }
 
+  renderVersionDisplay = () => {
+    const { pipelineRun } = this.props;
+    if (pipelineRun && pipelineRun.version && pipelineRun.version.pipeline) {
+      const versionString = `v${pipelineRun.version.pipeline}`;
+      const alignmentDBString = pipelineRun.version.alignment_db
+        ? `, NT/NR: ${pipelineRun.version.alignment_db}`
+        : "";
+
+      return versionString + alignmentDBString;
+    }
+    return "";
+  };
+
+  renderHeader() {
```

Comment:
Can you break it into `PipelineVizHeader` since this component is getting somewhat large?